src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -385,6 +387,12 @@ private ParseUserDefinitionResult ParseUDFsAndNamedFormulas(string script, Parse
                         definitionBeforeTrivia.Add(ParseTrivia());
                         var result = ParseExpr(Precedence.None);
 
+                        if (result is TypeLiteralNode _)
+                        {
+                            CreateError(equalToken, TexlStrings.ErrUserDefinedTypeIncorrectSyntax);

```

Comment:
>equalToken [](http://example.com/codeflow?start=40&length=10)

Can we simply call `_curs.TokCur` and remove the variable declaration at ln 365?

Reply:
we can't, if we want the error highlighting at equal token. `_curs.TokCur` will be different at this point as we do more parsing

Reply:
Got it.